style.scss

```diff
@@ -0,0 +1,98 @@
+body, h1 {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: rgb(226, 226, 226);
+}
+
+.container {
+    max-width: 25em;
+    margin: 1em auto;
+    background-color: white;
+    box-shadow: 0 0 10px rgb(172, 172, 172);
+}
+
+.todo {
+    margin-top: .5em;
+    padding-top: .5em;
+}
+
+.todo__content {
+    padding: 1em;
+}
+
+.todo__header {
+    text-align: center;
+}
+
+/* New Todo Entry Field */
+.todo__new-todo-field {
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    font-size: 1em;
+    padding: .5em;
+}
+
+.todo__complete-all-field {
```

Comment:
Avoid adding css class names in HTML when they don't actually do anything. Just leave it classless until you find you need to style something.